### DISCUSSION TITLE
assistment数据集问题花费时间的问题

### DISCUSSION THREAD

**Discussion by WHUTwyz** [pykt-team/pykt-toolkit#102] [Q&A] [answered]
据我所知，assistment2017有timetaken这一列属性，而assist09,12有overlap_time这一属性

**Comment by sonyawong** [comment, marked as answer]
Hi,感谢您对我们工作的关注. 在assistment2017的数据处理过程中，我们发现利用endTime-startTime并不一定等于timeTaken, 因此没有将timeTaken作为duration使用。同时assist2009虽然有overlap_time,但是官方也有该字段说明其计算经常出现错误“This field is often computed incorrectly. Many data sets display overlap time the same as the first response time.  You could compute overlap time using other fields, like using the state time of two problems.  “ 因此，我们当前实验都不采用其花费时间信息的字段。
